explorer/src/pages/AccountDetailsPage.tsx

```diff
@@ -217,6 +259,63 @@ function MoreSection({
       )}
       {tab === "history" && <TransactionHistoryCard pubkey={pubkey} />}
       {tab === "largest" && <TokenLargestAccountsCard pubkey={pubkey} />}
+      {tab === "vote-history" && data?.program === "vote" && (
+        <VotesCard voteAccount={data.parsed} />
+      )}
+      {tab === "slot-hashes" &&
+        data?.program === "sysvar" &&
+        data.parsed.type === "slotHashes" && (
+          <SlotHashesCard sysvarAccount={data.parsed} />
+        )}
+      {tab === "stake-history" &&
+        data?.program === "sysvar" &&
+        data.parsed.type === "stakeHistory" && (
+          <StakeHistoryCard sysvarAccount={data.parsed} />
+        )}
+      {tab === "blockhashes" &&
+        data?.program === "sysvar" &&
+        data.parsed.type === "recentBlockhashes" && (
+          <BlockhashesCard sysvarAccount={data.parsed} />
+        )}
     </>
   );
 }
+
+function getTabs(data?: ProgramData): Tab[] {
+  const tabs: Tab[] = [
+    {
+      slug: "history",
+      title: "History",
+      path: "",
+    },
+  ];
+
+  const programTypeKey = [
+    data?.program,
+    (data?.parsed as { type: any })?.type,
+  ].join(":");
```

Comment:
Typically you will want to use the `"property" in object` approach to isolate a property with proper type checking in TypeScript. The code you wrote is probably safe but if for some reason `parsed` could be a string, it would cause an error (the `in` syntax wouldn't compile if that was the case)

```suggestion
  let programTypeKey = "";
  if (data && "type" in data.parsed) {
    programTypeKey = `${data.program}:${data.parsed.type}`;
  }
```

Reply:
Looks a bit cleaner too.